src/container/rpc.h

```diff
@@ -24,7 +24,7 @@
  * These are for daos_rpc::dr_opc and DAOS_RPC_OPCODE(opc, ...) rather than
  * crt_req_create(..., opc, ...). See src/include/daos/rpc.h.
  */
-#define DAOS_CONT_VERSION 3
+#define DAOS_CONT_VERSION 4
```

Comment:
This change is okay. Just FYI, Johann's rule is that if we have already incremented this version since 1.2, then we don't need to increment it again before releasing 2.0.